payments_service/base/tests/__init__.py

```diff
@@ -29,6 +30,17 @@ def tolist(self, *args, **kw):
 
 class TestCase(DjangoTestCase):
 
+    def setUp(self):
+        super(TestCase, self).setUp()
+
+        # Since all endpoints pretty much use Solitude, set up the mock
+        # for convenience.
+        p = mock.patch('payments_service.solitude.api')
```

Comment:
sure, I added it and nothing failed. However, I doubt it will help much since curling/slumber rely so much on magic properties